<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandlerWindow</class>
 <widget class="QMainWindow" name="HandlerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NXM Handler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Use this list to configure programs to handle nxm links. Different Programs can be set up to handle links for different games. If the same game is supported by multiple binaries, the top-most is used.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="SortableTreeWidget" name="handlersWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>120</number>
      </attribute>
      <column>
       <property name="text">
        <string>Supported Games</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Binary</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Primary Handler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="handlerView">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="registerButton">
        <property name="text">
         <string>Register active</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../organizer/resources.qrc">
     <normaloff>:/MO/gui/remove</normaloff>:/MO/gui/remove</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove Binary</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../organizer/resources.qrc">
     <normaloff>:/MO/gui/add</normaloff>:/MO/gui/add</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add a binary</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SortableTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>sortabletreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../organizer/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>HandlerWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
